Resources/Prototypes/Catalog/Fills/Backpacks/duffelbag.yml

```diff
@@ -258,7 +258,7 @@
       - id: BoxSurvivalSyndicate
       - id: WeaponPistolViper
       - id: PinpointerNuclear
-
+      - id: AgentIDCard
```

Comment:
Why, this should be in the loadout if anything. From what i figured in the discussion it's because you couldnt get default access working on the card, but if this is the case then dont give em to them yet and make a bug report